slideShow.js

```diff
@@ -0,0 +1,116 @@
+// json取得先
+const fetchURL = 'https://jsondata.okiba.me/v1/json/Z96fg210419122813'
+
+// DOMの取得
+const slideShow = document.getElementById('js-slideShow')
+const slideShowContainer = document.getElementById('js-slideShowContainer')
+const prevArrow = document.getElementById('js-prevArrow')
+const nextArrow = document.getElementById('js-nextArrow')
+const pagination = document.getElementById('js-pagination')
+
+const slideState = {
+  currentNum: 0,
+  slideImageArray: []
+}
+
+let { currentNum } = slideState
+const { slideImageArray } = slideState
+
+const myFetch = async (fetchURL) => {
+  try {
+    const res = await fetch(fetchURL)
+    const data = await res.json()
+    return data
+  } catch {
+    alert('データを取得できませんでした。')
```

Comment:
自分はですけどアラートは使ったことないです

```suggestion
  } catch(e) {
  console.error(e)
   slideShowContainer.textContent = "データを取得できませんでした"
```

Reply:
修正しました！
786a952